### ISSUE TITLE
Incorrect indentation on line after { followed by comment

### ISSUE THREAD

**Issue by djlindstr** [Groovy-Emacs-Modes/groovy-emacs-modes#112]
The indentation error seems to be triggered by adding a comment after an opening brace. The following line is lined up to below the opening brace + 1.

```
whatever {
    indented_ok
}

whatever { // Comment
          indented_too_far
}

whatever { /* Comment */
          indented_too_far
}

{ -> /* Comment */
 indented_too_little
}
```

**Comment by Wilfred** [comment]
I think this was fixed in a6d50dc29601c3acfab211d7dc8699482844f290, are you running the latest version?

**Comment by djlindstr** [comment]
Indeed, my apologies. Thought I loaded the latest version. Keep up the good work!
